DataFormats/TrackingRecHitSoA/src/classes_def.xml

```diff
@@ -1,34 +1,34 @@
 <lcgdict>
-  <class name="TrackingRecHitSoA<pixelTopology::Phase1>"/>
-  <class name="TrackingRecHitSoA<pixelTopology::Phase1>::Layout<>"/>
-  <class name="TrackingRecHitLayout<pixelTopology::Phase1>"/>
-  <class name="TrackingRecHitSoAView<pixelTopology::Phase1>"/>
+  <class name="reco::TrackingRecHitSoA<pixelTopology::Phase1>"/>
+  <class name="reco::TrackingRecHitSoA<pixelTopology::Phase1>::Layout<>"/>
+  <class name="reco::TrackingRecHitLayout<pixelTopology::Phase1>"/>
+  <class name="reco::TrackingRecHitSoAView<pixelTopology::Phase1>"/>
 
-  <class name="PortableHostCollection<TrackingRecHitLayout<pixelTopology::Phase1>>"/>
-  <class name="TrackingRecHitHostPhase1" ClassVersion="3">
-    <version ClassVersion="3" checksum="3242244550"/>
+  <class name="PortableHostCollection<reco::TrackingRecHitLayout<pixelTopology::Phase1>>"/>
+  <class name="TrackingRecHitHostPhase1" ClassVersion="4">
+    <version ClassVersion="4" checksum="3602783515"/>
   </class>
```

Comment:
Given that `TrackingRecHitHostPhase1` itself is an instantiation of a class template, its version should not be tracked this way (but with `CMS_CLASS_VERSION()` in the class template definition). But given that schema evolution of `PortableHostCollection` is unexplored anyway (and therefore nothing is promised), I'd suggest to remove the class version for now (until the schema evolution is properly understood)
 ```suggestion
  <class name="TrackingRecHitHostPhase1"/>
```
(I might have given false advice on this in the earlier PR)

Reply:
@makortel would this also be the case for the tracks and vertices instances `TracksHostPhase1` and `ZVertexHost`? They also have the version tracked in the same way as the rechits [here](https://github.com/borzari/cmssw/blob/AlpakavsCUDA/DataFormats/TrackSoA/src/classes_def.xml) and [here](https://github.com/borzari/cmssw/blob/AlpakavsCUDA/DataFormats/VertexSoA/src/classes_def.xml)

Reply:
> would this also be the case for the tracks and vertices instances `TracksHostPhase1` and `ZVertexHost`?

Yes. `ZVertexHost` is already taken care of in https://github.com/cms-sw/cmssw/pull/43952 .

Reply:
> Yes. `ZVertexHost` is already taken care of in #43952 .

Alright. Will change it only for tracks

Reply:
Thanks!

Reply:
Modified for [rechits](https://github.com/cms-sw/cmssw/pull/43964/commits/a1aa8b8fd1fd83825254450bf6b88e33d4f60338#diff-5b9972ab207800790c7b8c2119734edee6182716b603bc30e397aed991cc3c3a) and [tracks](https://github.com/cms-sw/cmssw/pull/43964/commits/a1aa8b8fd1fd83825254450bf6b88e33d4f60338#diff-50ba4cf1f951631bdd446c3fb25bd3ad2af064c982b46b80261529b202547d61)